'yo botbuilder-assistant:skill' error "Cannot find module 'express-jwt'"

## Project

**Name**:  Skills

**Language**: TypeScript

## Description
Running 'yo botbuilder-assistant:skill' results in an exception on `tsc --p tsconfig.json && npm run coy-templates`

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! testskill-17@1.0.0 build: `tsc --p tsconfig.json && npm run copy-templates`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the testskill-17@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\v-eridah\AppData\Roaming\npm-cache\_logs\2019-05-22T18_44_17_049Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! testskill-17@1.0.0 postinstall: `npm run build && node ./deployment/webConfigPrep.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the testskill-17@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\v-eridah\AppData\Roaming\npm-cache\_logs\2019-05-22T18_44_17_111Z-debug.log

## To Reproduce
Install prereqs and run `yo botbuilder-assistant:skill`

## Expected behavior
No errors when using the skills generator.

## Screenshots
![image](https://user-images.githubusercontent.com/11055362/58200271-46b1d700-7c87-11e9-9b0b-d34955638797.png)


## Additional context
npm install @types/express-jwt fixes the issue, and `tsc --p tsconfig.json && npm run coy-templates` runs successfully afterwards.

Windows 10
Node 10.14.1
npm 6.4.1

--- a/lib/typescript/botbuilder-skills/package.json
+++ b/lib/typescript/botbuilder-skills/package.json
@@ -28,7 +28,7 @@
         "botframework-schema": "^4.4.0",
         "i18n": "^0.8.3",
         "jsonwebtoken": "^8.5.1",
-        "jwks-rsa": "^1.4.0",
+        "jwks-rsa": "1.5.0",
         "microsoft-bot-protocol": "^0.0.1",
         "microsoft-bot-protocol-websocket": "^0.0.1",
         "p-queue": "^4.0.0",

